api/PushSubscriptionChangeEvent.json

```
@@ -17,7 +17,7 @@
             "version_removed": "79"
           },
           "firefox": {
-            "version_added": false,
+            "version_added": "137",
             "notes": "The `pushsubscriptionchange` event is fired but does not have the `oldSubscription` and `newSubscription` properties. See [bug 1497429](https://bugzil.la/1497429)."
```

Let's update the existing `notes`:

```suggestion
            "notes": "Before Firefox 137, The `pushsubscriptionchange` event is fired, but does not have the `oldSubscription` and `newSubscription` properties. See [bug 1635524](https://bugzil.la/1635524)."
```